### ISSUE TITLE
The grid pagers are created using tables which have no headers

### ISSUE THREAD

**Issue by julie4754** [tonytomov/jqGrid#991]
The grid pagers are created using tables which have no headers and accessibility checkers flag this as a violation. Consider not using tables for the pager or introduce table headers.

Rule: Data table must identify headers

**Comment by tonytomov** [comment]
Hello,

Thanks. Will check this and replay ASAP.
